tests/unit/StandaloneDirectorsFiling.spec.ts

```diff
@@ -1318,10 +1333,10 @@ xdescribe('Standalone Directors Filing - Part 5 - Data', () => {
     expect(payload.filing).toBeDefined()
     expect(payload.filing.changeOfDirectors).toBeDefined()
 
-    let names = payload.filing.changeOfDirectors.directors.map(el => el.officer.firstName)
-    expect(names).toContain('Unchanged')
-    expect(names).toContain('Appointed')
-    expect(names).toContain('Ceased')
+    const firstNames = payload.filing.changeOfDirectors.directors.map(el => el.officer.firstName)
```

Comment:
This may seems like a frivolous change, but I stared at the old code for a while before figuring it out. I hope the new name makes more sense and will save someone else a couple minutes.